src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
> Probably more standard would be to put that OpenSeadragon function inside of the UMD module

True, see original discussion https://github.com/openseadragon/openseadragon/pull/984#discussion_r72284793 . To make it properly one would need to touch all source files. But it seems this is unnecessary because current solution provide proper isolation anyway.